draft-ietf-tls-esni.md

```diff
@@ -554,6 +565,9 @@ it does a standard ClientHello, with the exception of the following rules:
    allows servers that reject ECH and use ClientHelloOuter to safely ignore any
    early data sent by the client per {{RFC8446}}, Section 4.2.10.
 
+Note that these rules may change in the presence of an application profile
+specifying otherwise.
```

Comment:
No objection to the sentence, though I'm unsure what it's trying to do in the context of this PR.

Reply:
I threw this in to allow clients to omit the public name in CHO.

Reply:
If someone is allowed to change these rules, then how can they really be MUSTs?